webpack.config.js

```diff
@@ -1,14 +1,20 @@
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+
+/**
+ * WordPress dependencies
+ */
+const defaultConfig = require( '@wordpress/scripts/config/webpack.config' );
+
 module.exports = {
+	...defaultConfig,
 	entry: {
 		editor: './js/src/editor.js',
 	},
-	module: {
-		rules: [
-			{
-				test: /\.js$/,
-				loader: 'babel-loader',
-				exclude: /node_modules/,
-			},
-		],
+	output: {
+		path: path.resolve( process.cwd(), 'js', 'dist' ),
```

Comment:
This could be simplified to `path.resolve( __dirname, 'js/dist' )` per [this example](https://webpack.js.org/configuration/output/#outputpath).

Reply:
Ah, good idea. https://github.com/xwp/block-extend/pull/10/commits/b7a7288d1d0114f08031c0f3c9f86ab52a4a76d4 applies it.